### PR TITLE
docs: add English version of milestone.md

### LINKED ISSUE #1489: doc translation (Chinese to English): milestones
- Translate the Chinese version to English: https://openmldb.ai/docs/zh/v0.4/about/milestones.html  ((you can download as a markdown file from the button at the top-right corner))
- Save the English version to docs/en/milestones.md

### PR DESCRIPTION
**Pull request by cswjd**
docs: add English version of milestone.md 
close #1489

### CODE REVIEW

**Comment by cswjd** [comment]
fixed #1489

File: docs/en/milestones.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# OpenMLDB development history
```
Comment (lumianph) [inline docs/en/milestones.md]: The Evolution of OpenMLDB

File: docs/en/milestones.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## 1. From commercial closed source to community open source
```
Comment (lumianph) [inline docs/en/milestones.md]: capitalize the first letter of words for a title

File: docs/en/milestones.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+OpenMLDB originated as an internal commercial product of [Fourth Paradigm](https://www.4paradigm.com), an industry-leading artificial intelligence platform and service provider. The commercial AI platform provided by 4Paradigm covers the entire process of machine learning end-to-end application development and implementation, including data governance, feature engineering, model training and inference, and model management. Its products have been deployed and applied on a large scale in hundreds of enterprise-level scenarios.
```
Comment (lumianph) [inline docs/en/milestones.md]: Fourth Paradigm => 4Paradigm

File: docs/en/milestones.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+In 2021, the core development team of OpenMLDB will abstract, enhance, and add many community-friendly features to the core modules of data governance and feature engineering in the 4Paradigm internal commercial products. Today's open source project OpenMLDB. OpenMLDB is born out of a commercial product that has been tested for up to five years, and has a lot of experience and unique understanding in this field. Today, OpenMLDB develops based on the open source and open community, and hopes to help more enterprises complete the transformation of artificial intelligence with low cost and high quality.
```
Comment (lumianph) [inline docs/en/milestones.md]: In 2021, the core development team of OpenMLDB will abstract, enhance, and add many community-friendly features to the core modules of data governance and feature engineering in the 4Paradigm internal commercial products.
=>
In 2021, the development team abstracted, enhanced, and added many community-friendly features to the core modules of data governance and feature engineering in the commercial products, and finally made it publicly available as an opensource project OpenMLDB.

File: docs/en/milestones.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+## 2. Milestones
+
+### 2017.2 - 2021.5: Internal  MLDB
+
+- 2017.2: The first code commit
+- 2017.9: The first financial industry customer, China Guangfa Bank
+- 2019.8: The first retail industry user, Yum!
```
Comment (lumianph) [inline docs/en/milestones.md]: Yum! => Yum China

File: docs/en/milestones.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+- 2020.12: The cumulative number of landing scenes exceeds 100
```
Comment (lumianph) [inline docs/en/milestones.md]: landing scenes => real-world applications

File: docs/en/milestones.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+### 2021.6 - Today: Open Source OpenMLDB
+
+- 2021.6: Machine learning database OpenMLDB officially announced open source, providing enterprise-level FeatureOps full-stack solution
```
Comment (lumianph) [inline docs/en/milestones.md]: enterprise-level FeatureOps full-stack solution
enterprise-grade full-stack FeatureOps solution

File: docs/en/milestones.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+- 2021.7: v0.2.0 released, supports RestAPI access, optimized code style and comments
```
Comment (lumianph) [inline docs/en/milestones.md]: supports => supporting
 optimized code style and comments => code style and comments improved

File: docs/en/milestones.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+- 2021.8: OpenMLDB-based optimization innovation papers are published at VLDB 2021, the top international academic conference on databases
+- 2021.8: Recognized by Gitee and awarded the title of GVP (Gitee Most Valuable Project, the most valuable open source project)
```
Comment (lumianph) [inline docs/en/milestones.md]: delete ", the most valuable open source project"

File: docs/en/milestones.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+- 2021.9: The first open source community enterprise user, [Akulaku](https://www.akulaku.com/)
```
Comment (lumianph) [inline docs/en/milestones.md]: delete "open source"

File: docs/en/milestones.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+- 2021.9: Won the "Open Source Community and Open Source Project" award from the Academy of Information and Communications Technology
+- 2021.11: v0.3.0 released, supporting single-machine deployment mode for rapid deployment
```
Comment (lumianph) [inline docs/en/milestones.md]: single-machine deployment mode
=>
the standalone deployment

File: docs/en/milestones.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+- 2021.12: OpenMLDB Contributor Program launched
+- 2021.12: Host the first Feature Store Meetup in China
+- 2022.1: v0.4.0 is released, enhancing the development experience with SQL as the core, and introducing an online monitoring module
```
Comment (lumianph) [inline docs/en/milestones.md]: v0.4.0 released, enhancing the SQL-centric development experience, introducing the new online monitoring module

File: docs/en/milestones.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+- 2022.1: OpenMLDB first community meeting
+
+## 3. Roadmap
+
+The future development of OpenMLDB will firmly follow the path of community-based open source and openness, in order to benefit more enterprises and developers. In the future, in terms of technology, we will pay more attention to technologies that can effectively reduce the threshold and cost of use, such as cloud-native technology; in terms of community building, we will accelerate ecological construction, improve product information, and expand cooperation between industry and academia; finally, We hope that more developers and users can participate in our community and jointly promote the rapid growth of the OpenMLDB community.
```
Comment (lumianph) [inline docs/en/milestones.md]: delete "openness"
delete ", in order "

Comment (lumianph) [reply docs/en/milestones.md]: we will accelerate ecological construction, improve product information,
=>
OpenMLDB will integrate into the open-source ecosystem,

Comment (lumianph) [reply docs/en/milestones.md]: We hope => we hope

**Review by lumianph — CHANGES_REQUESTED** [review]

File: docs/en/milestones.md (hunk 13 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 12 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 11 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 10 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 9 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 8 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 7 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 6 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 5 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 4 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 3 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 2 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

File: docs/en/milestones.md (hunk 1 above)
Comment (cswjd) [reply docs/en/milestones.md]: Ok, thank you.

**Review by lumianph — APPROVED** [review]
LGTM, thanks

**Review by lumianph — APPROVED** [review]
LGTM

**Pull request merged**